workflow/Snakefile

```diff
@@ -40,45 +44,50 @@ def get_e2g_config(config):
 	e2g_config["E2G_DIR_PATH"] = E2G_DIR_PATH
 	e2g_config["ABC_BIOSAMPLES"] = config["ABC_BIOSAMPLES"]
```

Comment:
Did you not have to transform the tsv file into a separate format? the config_datasets.tsv has different columns than the typical ABC biosamples. I'm surprised you were able to get the workflow to run. Did the output look correct?

Reply:
config_datasets.tsv has the columns for ABC biosamples as well as a few additional ones, but the extra ones are just not referenced by ABC so it works fine.